tests/fixtures/nodes/src/nodes.rs

```diff
@@ -39,10 +39,14 @@ use tokio::runtime::Runtime;
 use tracing_fixture::{tracing, Tracing};
 use xshell::{cmd, Shell};
 
-const PAYER_FUND_CHUNK: usize = 20;
+// These can be overwritten by environment variables
+const TEST_MAX_PAYERS_NUM: usize = 20;
+const TEST_BALANCE_TOP_UP_AMOUNT_NIL: TokenAmount = TokenAmount::Nil(45); // this is how much it costs to run this test suite with prices set in `tests/resources/network/default/1.network.yaml` when running with 1 payer
```

Comment:
This used to be 1nil and now it's 45nil right? Why is this 45x what is was before? Can you also put the comment above this line rather than after it? This line is unnecessarily long

Reply:
This is the cost of running all func tests by 1 payer with current prices. Before we had 100 payers.
Also, this balance (in nilchain) is used only when where is no pre-funded balance in the leader DB (in remote func tests).

Reply:
But with default parameters the test uses max payers (20) * top up amount (45) = 900 nils right? Should this default be kept lower and let any environment that wants to use 1 payer adjust the amount accordingly?